nat-lab/bin/core-api.py

```diff
@@ -132,31 +170,94 @@ def add_node(self, req):
             public_key=req.public_key,
             os=req.os,
             os_version=req.os_version,
-            device_type=req.device_type,
-            nickname=req.nickname,
-            discovery_key=req.discovery_key,
-            relay_address=req.relay_address,
+            device_type=req.device_type or "",
+            nickname=req.nickname or "",
+            discovery_key=req.discovery_key or "",
+            relay_address=req.relay_address or "",
             traffic_routing_supported=req.traffic_routing_supported,
         )
 
         self.server.add_machine(node)
 
-        pprint(node)
         return node
 
     def handle_machines_delete(self):
         uid = self.path.removeprefix(self.machines_path)[1:]
-        print("uuid:", uid)
 
-        try:
-            self.server.remove_machine(uid)
-            print(f"{uid} removed")
-            self.send_response(204)
-        except KeyError:
-            print(f"{uid} unknown")
-            self.send_response(400)
+        if self.server.remove_machine(uid):
+            self._set_headers(status_code=204)
+        else:
+            self._send_error_response(101102, "Machine not found", 404)
```

Comment:
Maybe there's a library for the HTTP status codes we could use? They are pretty descriptive in other languages as an enum, would be nice to use those instead of just numbers. This would reduce the cognitive load slightly :)

Reply:
I see usage of HTTP_STATUS :rocket: